felles/src/main/kotlin/no/nav/familie/kontrakter/felles/tilbakekreving/Brevmottaker.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.familie.kontrakter.felles.tilbakekreving
+
+import javax.validation.Valid
+import javax.validation.constraints.Pattern
+
+data class Brevmottaker(
+    val type: MottakerType,
+    val vergetype: Vergetype? = null,
+    @field:Pattern(regexp = "^(.{1,80})\$", message = "Feltet kan ikke være tomt eller innholde mer enn 80 tegn")
+    val navn: String,
+    @field:Valid
+    val manuellAdresseInfo: ManuellAdresseInfo
+)
```

Comment:
Burde vi hatt med noe sånt som validerer at vergetype er spesifisert når mottakertypen er verge eller fullmektig?
```suggestion
) {
    init {
        if (type == MottakerType.VERGE || type == MottakerType.FULLMEKTIG) requireNotNull(vergetype)
    }
}```

Reply:
skader jo ikke det.